common/djangoapps/student/models.py

```diff
@@ -251,7 +251,11 @@ class Meta:  # pylint: disable=missing-docstring
     goals = models.TextField(blank=True, null=True)
     allow_certificate = models.BooleanField(default=1)
     bio = models.TextField(blank=True, null=True)
-    has_profile_image = models.BooleanField(default=0)
+    profile_image_version = models.CharField(max_length=8, null=True)
```

Comment:
balancing the size of the stored version data with the size of the resulting url, which i actually think could be even shorter than i have it now (down to 4 chars from 8).  i never thought about parsing the timestamp from the url and using it like that...hm.  i don't think it's a requirement, and it's also available by looking at the create time of the file in storage, so i'm kinda reluctant to commit to an extra bit of 'api' that seems like YAGNI.